src/content/docs/resources/faq.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Frequently Asked Questions (FAQ)
+description: Frequently Asked Questions
+---
+
+Welcome to Thoth Tech's Frequently Asked Questions (FAQ) section! We understand that you might have
+queries about contributing to Thoth Tech or seek clarification on how to use certain tools. Our aim
+is to provide you with comprehensive and straightforward answers to address your concerns promptly.
+
+<details>
+  <summary>Where can I find the link to Thoth Tech GitHub channel?</summary>
+
+Find the link [here](https://github.com/thoth-tech/).
+
+</details>
+
+<details>
+  <summary>Which Thoth Tech repository should I clone to contribute to the Documentation Website?</summary>
+
+To contribute to the Documentation Website, locate the
+["Thothtech-Documentation-Website" repository](https://github.com/thoth-tech/ThothTech-Documentation-Website/tree/main),
+navigate to the `src/content/docs`, and upload your `.md` or `.mdx` file to the relevant folder.
+
+</details>
+
+<details>
+  <summary>When I attempt to run the documentation website locally, it says "The .nvmrc file is using a version of node that is not supported by Astro anymore". How do I fix this?</summary>
+
+Re-install or upgrade the version of Node.js software on your laptop.
+
+</details>
+
+<details>
+  <summary>How do I make a newly created folder appear on the side navbar of the Documentation website?</summary>
+
+In the `astro.config.mjs` file, create a new label (if it’s a main folder) or item (if it’s a subfolder). 
+You do not need to reinvent the wheel; you can copy from what already exists in the `astro.config.mjs` file, 
+insert the copied block into the right location, and modify the labels and directory link.
+
+</details>
+
+<details>
+  <summary>Is there a video that helps me understand how to migrate documents to the Documentation website?</summary>
+
+Yes, there is. Find some helpful links below:
+
+- [Instructional video](https://deakin365.sharepoint.com/:v:/r/sites/ThothTech2/Shared%20Documents/General/Instructional%20Video%20-%20How%20To%20Upload%20Documents%20to%20Starlight.mp4?csf=1&web=1&e=s26lMo).
+- [Help hub session recording](https://deakin365-my.sharepoint.com/:v:/g/personal/s221460489_deakin_edu_au/EXib27xJI2hCkHpzBXOxxYQBOfDGhhQJ1A23ciisMROeRA?xsdata=MDV8MDF8fDgxMTZlNDYzMzAyZTQxYTZhNGE2MDhkYmYyMWMwMGY1fGQwMjM3OGVjMTY4ODQ2ZDU4NTQwMWMyOGI1ZjQ3MGY2fDB8MHw2MzgzNjk5NzQyOTIxNjE3MDJ8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxMM1JsWVcxekx6RTVPbEZtZUY5VFZFaFZPVEJQYzFaWlFraFdXVXRvYzFKUk5XZHRSV1V3Y3psUk5tdFBjRUptTm1Kc2FUZ3hRSFJvY21WaFpDNTBZV04yTWk5amFHRnVibVZzY3k4eE9UcFJabmhmVTFSSVZUa3dUM05XV1VKSVZsbExhSE5TVVRWbmJVVmxNSE01VVRaclQzQkNaalppYkdrNE1VQjBhSEpsWVdRdWRHRmpkakl2YldWemMyRm5aWE12TVRjd01UUXdNRFl5T0RNd01nPT18NTg0ZjIwNWZmZjA3NGZlZTQyNjQwOGRiZjIxYzAwZjN8ZTU2N2Y4NzA4YjFlNDI1YWIzYjRhM2I1NmExY2QwMjE%3D&sdata=TldtNGVnTDJWWlVnb2lBMGlZZXlLNEI1M2p5emU3ZndBYmpSTkV6MnN0QT0%3D).
```

Comment:
I don't have access to this video, is this just a permission issue for me or can anyone else view this video?

Reply:
I can't access that video either

Reply:
I've fixed this.

There was a problem where the link from Teams (i.e. Sharepoint) would return a 404 or permission error.

I've migrated this video over to Panopto which should now work.